docs/first_workspace.md

```diff
@@ -0,0 +1,185 @@
+# Making a Pixi workspace
+
+Pixi's biggest strength is its ability to create reproducible, powerful, and flexible workspaces.
+Let's go over the common steps to create a simple Pixi workspace.
+
+## Creating a Pixi workspace
+To create a new Pixi workspace, you can use the `pixi init` command:
+
+```shell
+pixi init my_workspace
+```
+
+This command creates a new directory called `my_workspace` with the following structure:
+
+```shell
+my_workspace
+├── .gitattributes
+├── .gitignore
+└── pixi.toml
+```
+
+The `pixi.toml` file is the manifest of your Pixi workspace.
+It contains all the information about your workspace, such as its channels, platforms, dependencies, tasks, and more.
+
+The file created by `pixi init` is a minimal manifest that looks like this:
+
+```toml title="pixi.toml"
+[workspace]
+authors = ["Jane Doe <jane.doe@example.com>"]
+channels = ["conda-forge"]
+name = "my_workspace"
+platforms = ["osx-arm64"]
+version = "0.1.0"
+
+[tasks]
+
+[dependencies]
+```
+
+??? tip "Do you want autocompletion of the manifest file?"
+    As `pixi.toml` has a JSON schema, it is possible to use IDE’s like VSCode to edit the field with autocompletion.
+    Install the [Even Better TOML VSCode extension](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml) to get the best experience.
+    Or use the integrated schema support in PyCharm.
+
+## Managing dependencies
+After creating the workspace, you can start adding dependencies.
+Pixi uses the `pixi add` command to add dependencies to a workspace.
+This command will, by default, add the [**conda**](https://prefix.dev/blog/what-is-a-conda-package) dependency to the `pixi.toml`, solve the dependencies, write the [lock file](./workspace/lockfile.md), and install the package in the environment.
+For example, let's add `numpy` and `pytest` to the workspace.
+
+```shell
+pixi add numpy pytest
+```
+This results in these lines being added:
+
+```toml title="pixi.toml"
+[dependencies]
+numpy = ">=2.2.6,<3"
+pytest = ">=8.3.5,<9"
+```
+
+You can also specify the version of the dependency you want to add.
+
+```shell
+pixi add numpy==2.2.6 pytest==8.3.5
+```
+
+### PyPI dependencies
+Pixi normally uses `conda` packages for dependencies, but you can also add dependencies from [PyPI](https://pypi.org).
+Pixi will make sure it doesn't try to install the same package from both sources, and avoid conflicts between them.
+
+If you want to add them to your workspace you can do that with the `--pypi` flag:
+
+```shell
+pixi add --pypi requests
+```
+This will add the `requests` package from PyPI to the workspace:
+
+```toml title="pixi.toml"
+[pypi-dependencies]
+requests = ">=2.31.0,<3"
```

Comment:
Let's use the httpx package here: https://blog.ian.stapletoncordas.co/2024/02/a-retrospective-on-requests